.github/workflows/ci.yml

```diff
@@ -220,7 +220,7 @@ jobs:
       - name: Install packages
         run: |
           sudo apt-get update
-          sudo apt-get install build-essential libtool autotools-dev automake pkg-config bsdmainutils curl ca-certificates ccache python3 rsync git procps bison
+          sudo apt-get install build-essential libtool autotools-dev automake pkg-config bsdmainutils curl ca-certificates ccache python3-dev rsync git procps bison
```

Comment:
please only add `python3-dev` where we need it (in the matrix for those jobs where we run the `qa/` tests, because we shouldn't need it for anything else)

Reply:
have reverted back to python3.